openlibrary/components/IdentifiersInput.vue

```diff
@@ -115,40 +115,48 @@ export default {
     },
 
     computed: {
-        popularEditionConfigs: function() {
-            if (this.edition_popular) {
-                const popularConfigs = JSON.parse(decodeURIComponent(this.edition_popular));
-                return Object.fromEntries(popularConfigs.map(e => [e.name, e]));
+        parsedConfigs: function() {
```

Comment:
We generally use `config` singular.

```suggestion
        parsedConfig: function() {
```